CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+### Changed
+- Switched from Glide to Go Modules
```

Comment:
```suggestion
- Switched from Glide to Go Modules.
```

Reply:
<3 like this review (really do!) - will pay more attention.